main.go

```diff
@@ -42,18 +46,24 @@ func main() {
 			log.Fatalf("expected line to have 2 elements, got %d for %v", len(line), line)
 		}
 
-
 		quiz = append(quiz, item{
 			Question: line[0],
 			Answer:   line[1],
 		})
 	}
 
-	resultsJSON, err := json.MarshalIndent(quiz, "", "    ")
+	possibleScore := len(quiz)
+	actualScore := 0
 
-	if err != nil {
-		log.Fatal(err)
+	for i, item := range quiz {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Printf("Problem #%v: %s = ", i, item.Question)
+		text, _ := reader.ReadString('\n')
```

Comment:
Don't ignore errors.

Reply:
Gotcha. Repeat I will never ignore errors.

Reply:
Or copy code from SO and not catch this.. :)

Reply:
> Or copy code from SO 

Just end the sentence there